### PR TITLE
update installation doc

### LINKED ISSUE #479: MMCV Installation modification
**Describe the feature**

Modify the mmcv install installation document

**Motivation**

The latest MMCV can not be directly installed for some versions

**Related resources**

https://github.com/open-mmlab/mmcv/pull/748/files

**Additional context**

This one is proposed by members of MMCV

### PR DESCRIPTION
**Pull request by congee524**
update installation doc
Fixes #479

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ CFLAGS="${CFLAGS} -mavx2" pip install --upgrade --no-cache-dir --force-reinstall
 conda install -y jpeg libtiff
 ```
 
-**Note**:  You need to run `pip uninstall mmcv` first if you have mmcv installed.
+**Note**:  You need to run `pip uninstall mmcv (mmcv-full)` first if you have mmcv (mmcv-full) installed.
```
Comment (innerlee) [inline docs/install.md:46]: Does this mean that the user need to re-install mmcv-full?

Comment (congee524) [reply docs/install.md:46]: If the user intends to install mmcv, then mmcv-full should be uninstalled firstly.
Or it can be explained in more detail here

Comment (innerlee) [reply docs/install.md]: Please explain it with more detail

Comment (congee524) [reply docs/install.md]: Maybe the current version is enough to understand? (I have pushed a new commit half an hour ago)

Comment (innerlee) [reply docs/install.md]: mmcv-full is preferred for this repo, so let's ignore the replacing of `mmcv-full` with `mmcv` case.

**Review by innerlee — APPROVED** [review]

File: docs/install.md (hunk 1 above)
Comment (congee524) [reply docs/install.md]: ok

**Pull request merged**